src/utils/__tests__/scroll-test.ts

```diff
@@ -27,34 +33,46 @@ describe('scrollToLocation', () => {
             container.scrollLeft = left || 0;
             container.scrollTop = top || 0;
         });
+
+        // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+        // @ts-ignore
+        container.getBoundingClientRect = jest.fn(() => ({
+            height: 200,
+            left: 0,
+            top: 0,
+            width: 200,
+        }));
+
+        container.scrollLeft = 0;
+        container.scrollTop = 0;
     });
 
     test.each`
-        parentTop | scrollTop
+        offsetTop | scrollTop
         ${-200}   | ${0}
         ${0}      | ${0}
-        ${200}    | ${200}
-        ${2000}   | ${2000}
+        ${200}    | ${100}
```

Comment:
This tells me the behavior of the scroll is changing to scroll to the top of the page instead of the bottom, but I'm not sure why. Is that desirable?

Reply:
My understanding is that given the dimensions of the container (200x200), an element inside the container with `offsetTop` of -200 and 0 would have no need to apply any `scrollTop` because it would already be visible with `scrollTop` of 0